Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -114,25 +117,41 @@ def test_instance_connection(self) -> None:
         their own connectivity test
         """
 
-    def http_request_with_access_token(self, method: str, headers: Dict[str, str] | None = None, url_suffix='', params=None,
-                                       data=None, json_data=None, resp_type='json', ok_codes=None, full_url='',
-                                       files: Dict[str, Any] | None = None) -> Any:
-        """This method wraps the _http_request that comes from the BaseClient class, and adds the access_token
-        of the client to the headers request, by calling the get_access_token method, which is an abstract method,
-        and every child class must implement it.
+    def http_request(self, method: str, headers: dict[str, str] = {}, url_suffix='', params=None,
```

Comment:
Adding a default empty dictionary for an argument, **headers** in our case, can cause unwanted behavior, as explained [here](https://stackoverflow.com/questions/366422/how-can-i-avoid-issues-caused-by-pythons-early-bound-default-parameters-e-g-m), where the headers object can "remember" old data.